content/docs/1_guide/2_read-me/guide.txt

```diff
@@ -0,0 +1,250 @@
+Title: Read me first
```

Comment:
TBH I find it a bit weird that the title says "read me first" but this guide section is the second section after the quickstart guide.

The order makes sense to me (maybe even as third after the "Exploring the Starterkit" guide?). But the title could be something like "About the docs" to avoid a mental conflict between the quickstart and this guide.